### ISSUE TITLE
OpenSSL version mismatch

### ISSUE THREAD

**Issue by Natalia-T** [Voice-Privacy-Challenge/Voice-Privacy-Challenge-2024#6]
Bug happens while running ` 01_download_data_model.sh`:

`OpenSSL version mismatch. Built against 30000020, you have 30200010`

In the first example, “ldd /usr/bin/sftp” is run locally to see which libraries are loaded during the sftp call:

```
user@w7:/s/work/Voice-Privacy-Challenge-2024$ ldd /usr/bin/sftp
        linux-vdso.so.1 (0x00007ffd9955f000)
        libcrypto.so.3 => /lib/x86_64-linux-gnu/libcrypto.so.3 (0x00007fc4aa384000)
        libedit.so.2 => /lib/x86_64-linux-gnu/libedit.so.2 (0x00007fc4aa34a000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007fc4aa121000)
        libtinfo.so.6 => /lib/x86_64-linux-gnu/libtinfo.so.6 (0x00007fc4aa0ef000)
        libbsd.so.0 => /lib/x86_64-linux-gnu/libbsd.so.0 (0x00007fc4aa0d7000)
        /lib64/ld-linux-x86-64.so.2 (0x00007fc4aa807000)
        libmd.so.0 => /lib/x86_64-linux-gnu/libmd.so.0 (0x00007fc4aa0c8000)
```
In the second example, the same command (`“ldd /usr/bin/sftp”` ) is called after env activation inside `01_download_data_model.sh`. We can see that in the first case, all libraries are loaded from the system directories, and in the second case, there are two libraries (libcrypto & libtinfo) that are loaded from conda env, that causes the version conflict.  This bug will not appear if conda and system has the same libraries' versions.

```
user@w7:/s/work/Voice-Privacy-Challenge-2024$ ./01_download_data_model.sh
WARNING: No ICDs were found. Either,

Install a conda package providing a OpenCL implementation (pocl, oclgrind, intel-compute-runtime, beignet) or 

Make your system-wide implementation visible by installing ocl-icd-system conda package.
      linux-vdso.so.1 (0x00007ffcea784000)
      libcrypto.so.3 => /s/work/Voice-Privacy-Challenge-2024/venv/lib/libcrypto.so.3 (0x00007f3266d5a000)
      libedit.so.2 => /lib/x86_64-linux-gnu/libedit.so.2 (0x00007f3266d0a000)
      libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f3266ae1000)
      libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f3266adc000)
      libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f3266ad7000)
      libtinfo.so.6 => /s/work/Voice-Privacy-Challenge-2024/venv/lib/libtinfo.so.6 (0x00007f3266a94000)
      libbsd.so.0 => /lib/x86_64-linux-gnu/libbsd.so.0 (0x00007f3266a7c000)
      /lib64/ld-linux-x86-64.so.2 (0x00007f32672bf000)
      libmd.so.0 => /lib/x86_64-linux-gnu/libmd.so.0 (0x00007f3266a6f000)
```

I think this bug happens because `sshpass` was installed in the conda.

**Comment by pchampio** [comment]
`micromamba install OpenSSH --channel conda-forge` before running `01_download_data_model.sh` will fix the error, I'm putting it in the 00_install.sh.
Not sure if this is caused by sshpass.

```log
venv $~ ldd ./venv/bin/sftp
        linux-vdso.so.1 (0x00007fff80df2000)
        libedit.so.0 => /home/pchampio/lab/Voice-Privacy-Challenge-2024/./venv/bin/../lib/libedit.so.0 (0x00007f4e9405c000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f4e93e50000)
        libncurses.so.6 => /home/pchampio/lab/Voice-Privacy-Challenge-2024/./venv/bin/../lib/./libncurses.so.6 (0x00007f4e93e26000)
        libtinfo.so.6 => /home/pchampio/lab/Voice-Privacy-Challenge-2024/./venv/bin/../lib/./libtinfo.so.6 (0x00007f4e93de5000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f4e940ca000)
```

**Comment by pchampio** [comment]
Should be fixed by: 1a4ee437e56c2c697ddb244f416a048c7d2878c0
Please close the issue if this was the correct fix.
